models/src_klaviyo.yml

```diff
@@ -57,7 +57,7 @@ sources:
           - name: updated
             description: Timestamp of when the campaign was last updated.
         
-      - name: event
+      - name: event #Should we add more rigid freshness tests for the event table?
```

Comment:
that's a good question...i'm not sure how rigid to go, but maybe we could get some data on how often folks are syncing klaviyo and how long the syncs take. i'm tempted to just change it to 24 or 48 hours tho

Reply:
I would be interested to ask a customer how often they are syncing; however, I assume it is pretty regular with the events table. I also have noticed a few other customers have expressed the desire to adjust our freshness tests. 

This may be something to bring up with @fivetran-kristin during our standup to settle on an approach.